vlan: Add vlan package

terasawa-lin opened the pull request:
Modifications to the recipe

remove:
SRC_URI
SRC_URI[md5sum]
SRC_URI[sha256sum]
UPSTREAM_CHECK_URI
UPSTREAM_CHECK_REGEX
S
do_configure_append()
do_compile()

change:
LIC_FILES_CHKSUM

# Purpose of pull request

This pr adds vlan package based on meta-openembedded/meta-networking/recipes-connectivity/vlan/vlan_1.9.bb

# Test
## How to install
Add the following to conf/local.conf.
```
IMAGE_INSTALL_append = " vlan"
```
## How to test

Run following command from console

```
# vconfig add eth0 5
# ip addr add <IP address> dev eth0.5
# ip link set eth0.5 up
```

If the command is successful, eth0.5 will be added to the network device.

## Test result

```
# ifconfig
・・・
eth0.5    Link encap:Ethernet  HWaddr 36:90:E7:20:BB:79
          inet addr:<IP address>  Bcast:0.0.0.0  Mask:255.255.0.0
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
・・・
```

hiraku-wfs commented on recipes-debian/vlan/vlan_debian.bb:
```diff
@@ -0,0 +1,25 @@
+require recipes-debian/sources/vlan.inc
+
+SUMMARY = "VLAN provides vconfig utility"
+HOMEPAGE = "http://www.candelatech.com/~greear/vlan.html"
```
This URL is now invalid now. Instead, please refer to the following.
https://salsa.debian.org/debian/vlan

hiraku-wfs commented on recipes-debian/vlan/vlan_debian.bb:
```diff
@@ -0,0 +1,25 @@
+SUMMARY = "VLAN provides vconfig utility"
+HOMEPAGE = "http://www.candelatech.com/~greear/vlan.html"
+SECTION = "misc"
+LICENSE = "GPLv2"
```
Please see the "Copyright File" from the following page.
https://packages.debian.org/buster/misc/vlan

The file says this is GPLv2+.

terasawa-lin committed:
vlan: Add vlan package

Modifications to the recipe

remove:
SRC_URI
SRC_URI[md5sum]
SRC_URI[sha256sum]
UPSTREAM_CHECK_URI
UPSTREAM_CHECK_REGEX
S
EXTRA_OEMAKE
do_configure_append()
do_compile()

change:
HOMEPAGE
LICENSE
LIC_FILES_CHKSUM

terasawa-lin commented:
Fixed HOMEPAGE and LICENSE.
EXTRA_OEMAKE has been removed.

hiraku-wfs commented:
Thanks!
